src/connector/src/sink/file_sink/opendal_sink.rs

```diff
@@ -138,12 +145,19 @@ impl<S: OpendalSinkBackend> TryFrom<SinkParam> for FileSink<S> {
         let config = S::from_btreemap(param.properties)?;
         let path = S::get_path(config.clone()).to_string();
         let op = S::new_operator(config.clone())?;
+        let mut batching_strategy = S::get_batching_strategy(config.clone());
+        // If no batching strategy is defined, the default batching strategy will be used, which writes a file every 10 seconds.
+        if batching_strategy.max_row_count.is_none() && batching_strategy.rollover_seconds.is_none()
```

Comment:
Think twice, I suggest do not give a default `rollover_seconds`, but only give a default value explicitly when no other conditions are set. As no batching strategy means no `max_row_nums` and no `rollover_seconds` and possible additions in the future.
eg.  if we set default = 10s instead of Option<u64>, If the user sets` max_row_nums=10000`, a file will still be written every 10 seconds.